inventory/osinfo/osinfo.go

```diff
@@ -28,7 +28,7 @@ type DistributionInfo struct {
 }
 
 // Architecture attempts to standardize architecture naming.
-func Architecture(arch string) string {
+var Architecture = func(arch string) string {
```

Comment:
Why us this a var? We should use the actual implementation in tests as it does nothing fancy

Reply:
yeah... i did this initially thinking i might need to get mock output from architecture function; but it seems i dont need to mock the output of this method.